src/api/comment/content-types/comment/lifecycles.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+
+    async afterCreate(event) {
+
+        const ideaCard = event.result.idea_card
+        const ideaName = event.result.idea_card.ideaName
+        const commentAuthor = event.result.author
+        const commentText = event.result.text
+        const timeCreated = event.result.createdAt
+
+        await strapi.entityService.create('api::notification.notification', {
+            data: {
+              Title: commentAuthor + " commented on " + ideaName,
+              Content: commentText,
+              Tag: "New Comment",
+              idea_card: ideaCard.id,
+              TimeCreated: timeCreated,
+              Read: false,
```

Comment:
how does the state changes from unread to read? are we supposed to send some sort of request to backend?

Reply:
My guess is it's going to be an attribute in the notification collection but correct me if I'm wrong @oriyomibadmus .

btw happy new year! :smile: